src/PAModel/Utility/Utilities.cs

```diff
@@ -148,16 +148,19 @@ public static JsonElement ToJson(this ZipArchiveEntry e)
         // returns "bar\hi.txt"
 
         // Net Core 2.1 provides a Path.GetRelativePath api
-        // but since we target netstandard, we can convert to URIs and make the relative path from that
+        // but since we target netstandard 2.0, we can convert to URIs and make the relative path from that
         // see https://stackoverflow.com/questions/275689/how-to-get-relative-path-from-absolute-path
-        public static string GetRelativePath(string full, string basePath)
+        // For reference, see Core's impl at: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/IO/Path.cs#L861
+        public static string GetRelativePath(string fullPathFile, string basePathDirectory)
         {
-            Uri fromUri = new Uri(AppendDirectorySeparatorChar(basePath));
-            Uri toUri = new Uri(AppendDirectorySeparatorChar(full));
+            // First arg is always a path name, 2nd arg is always a directory.
+            // directory is always a prefix. 
+            Uri fromUri = new Uri(AppendDirectorySeparatorChar(basePathDirectory));
+            Uri toUri = new Uri(fullPathFile);
```

Comment:
Knowing that the 1st param is always a file lets us simplify the expression and remove the pesky Path.HasExtension() check below - which was the root of problems in the bug.